### ISSUE TITLE
Data Specification for Flu (& Other Vaccination) Locations

### ISSUE THREAD

**Issue by mheadd** [codeforamerica/project-ideas#39]
Last year, the City of Philadelphia developed a [data specification](https://github.com/CityOfPhiladelphia/flu-shot-spec) for cities that wanted to publish flu shot locations. Philly, Chicago and SF all ended up using it to publish flu shot locations last year.

No idea if anyone will opt to use it this year, and into the future. Would be great to see stewardship of this data specification taken over by CfA.

**Comment by fureigh** [comment]
[Long Beach, Calif.](http://www.lbcflushots.org) and [Los Angeles](http://datala.github.io/flushots/) are using it this year. What does stewardship of the data specification entail?

**Comment by greggish** [comment]
Looks like it's not all that far from <a href="github.com/codeforamerica/OpenReferral">the Open Referral specification</a> ~ flu shots are a service provided at places! Could there be alignment here?

**Comment by mheadd** [comment]
Hmm, that actually might make sense - I see a lot of overlap between service/locations in Open Referral and the flu shot spec. Additionally, the flu shots spec should cover other kinds of inoculations as well, so maybe it is better incorporated into OR.

What would need to be done to fold it into OR?

**Comment by mheadd** [comment]
Several of the cities that have adopted the flu shot location spec - Philly, Chicago, NYC, San Francisco - had input into the current version. I would think that we would want to solicit their thoughts on the possibility of folding this into OR.

**Comment by greggish** [comment]
Right. It's a good use case. Let me know how i can help with this.

**Comment by greggish** [comment]
Okay. We already have adoption of OR in chicago and San Francisco, and I've
spoken with many in NYC.  I'd be happy to make a presentation, and/or
recruit someone to help with gap analysis -- let me know where/when.

**Comment by mheadd** [comment]
Looping in @tkompare who's built an app that uses data that conforms to the current flu shot spec format.

**Comment by mheadd** [comment]
@greggish - I'm happy to help with gap analysis and to take part in any discussions with cities.

**Comment by greggish** [comment]
Awesome!

FYI, the Open Referral spec is currently in v0.4 -- @spara is driving it to
1.0 by January. There will be a formal comment period along the way (early
December I believe) so there's the possibility for, say, tweaking based on
insights that you all have gleaned from deployment.
